data/transport missions.txt

```diff
@@ -1547,6 +1547,108 @@ mission "Shady passenger transport 3"
+			`	The remaining three goons attempt to flee, but your fingers find the button controlling the bridge door and it slams down before they can escape. They throw down their weapons and surrender. You toss them into the brig and decide that it might be a hoot to take them to their original destination as prisoners instead of passengers, and see if you can get a bounty for bringing them in.`
+
+			label negotiate
+			`"You don't want my ship, you want my money," you stammer, stalling for time while you try to think of a way out of this situation. "This ship is registered to me; nobody would believe that you came across it honestly. You want a fresh start in an anonymous ship. I'll give you 4 million credits, and you can buy a Modified Argosy or some other vessel better suited to your line of work."`
+			` The gang leader snorts derisively. "Why don't we just kill you and hotwire this ship? That sounds more fun."`
+			choice
+				`(Shoot first!)`
+					goto shoot
+				`(Offer more money.)`
+					goto "more money"
+				`(Surrender.)`
+					goto betrayed
+
+			label "more money"
+			`You continue: "You'll have a heck of a time taking over the ship without casualties. My bank account can't be accessed without my permission, and you know it. Use violence and you'll still be poor - and some of you might be dead. I'll give you 8 million, and that's my final offer."`
+			`	8 million credits is a lot of money. Thankfully, it's enough. A sigh of relief escapes your lips as everyone lowers their weapons.`
+			`	You arrange for a merchant captain to transport the renegades down to the surface, along with a password-protected credit chit. Once the airlock closer, you give the password. A lump appears in your throat as your communicator buzzes, alerting you to an unusually large transaction. At least you have escaped with your life and your ship. You are now free to continue your career--poorer, but hopefully wiser.`
+				defer
+
+			label betrayed
+			`You go to the security station and bitterly hand over the ship's access codes.`
+			`	"Smart move," the leader says. "Or was it? Now we don't need you alive for anything, do we?"`
+			`	She fires her shotgun into you from behind, and for a brief moment, you are treated to the sight of your internal organs splattered all over the console. It's the last thing you ever see.`
+				die
+
+	on complete
+		dialog `You hand off your prisoners to the planetary authorities who gratefully take them into custody. It turns out that they were wanted, and there's a substantial bounty. You find yourself richer to the tune of <payment>.`
+		payment 250000
+
+	on defer
+		payment -8000000
+
+	npc
+		government "Bounty Hunter"
+		personality nemesis plunders waiting
```

Comment:
should these be `staying`? Waiting has no meaning here since they are guaranteed to be in a different system when you take off.